ewaste/ewasteapp/migrations/0002_auto_20210726_1724.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.6 on 2021-07-26 17:24
```

Comment:
Delete this file. Make sure to migrate to the migration prior to this one before deleting.